metagraph/src/graph/alignment/dbg_aligner.hpp

```diff
@@ -139,96 +148,94 @@ class DBGAligner : public SeedAndExtendAligner<Seeder, Extender> {
 };
 
 
-template <class Seeder, class Extender>
-inline void SeedAndExtendAligner<Seeder, Extender>
+template <class ThisSeeder, class ThisExtender>
+inline void SeedAndExtendAligner<ThisSeeder, ThisExtender>
```

Comment:
I did this to prevent these from shadowing the abstract parent class `Seeder`. I fixed this by renaming the parent classes to `ISeeder` and `IExtender`